lib/commands/setup.js

```diff
@@ -1253,9 +1260,22 @@ SetupScreens.prototype.app = function app(callback) {
  * @param {Function} callback - Function to be called when the prompting finishes
  */
 SetupScreens.prototype.network = function app(callback) {
+	var defaultProxy = '';
+	if(this._config.get('cli.httpProxyServer')) {
+		defaultProxy = this._config.get('cli.httpProxyServer');
+	} else if(proxy.length > 0) {
+		var pI = 0, pL = proxy.length;
+		for(; pI < pL; pI++) {
```

Comment:
Typically I would just do:

~~~
for (var i = 0, len = proxy.length; i++) {
~~~

`pI` and `pL` are horrible names.

If you're feeling functional, you could do:

~~~
proxy.some(function (p) {
    if (p && p.valid) {
        defaultProxy = p.fullAddress;
        console.log(defaultProxy);
        return true;
    }
});
~~~